### DISCUSSION TITLE
OpenGL Support when?

### DISCUSSION THREAD

**Discussion by juamnitruh** [rafaelvcaetano/melonDS-android#976] [Q&A]
Honestly, that's the most important thing this emulator needs, more than multiplayer, at least imo.
I mean, sprite games run fine, but for games like New Super Mario Bros, Dragon Quests 4-6, and others, it suffers to run it in my device. JIT isn't enough.
Adding opengl would help so much and support more devices.

**Comment by DatEmulationGuy** [comment]
Ain't Google going to kill opengl? I think Vulkan would be better if that's the case

**Reply by Dwaqter** [reply]
A maior parte das pessoas possuem apenas opengl e adiciona-la seria uma otima adição, mas fica com o desenvolvedor decidir colocar ou não.

**Reply by Bardock88** [reply]
> A maior parte das pessoas possuem apenas opengl e adiciona-la seria uma otima adição, mas fica com o desenvolvedor decidir colocar ou não.

The OpenGL graphic API is being developed, as it is here: [list of things to resolve](https://github.com/rafaelvcaetano/melonDS-android#what-is-missing), Vulkan may be in the works as well. Regarding the graphics API part, it's always good to have options to choose from when playing, but it takes time and also study to be implemented.
